srcpkgs/coreboot-utils/template

```diff
@@ -0,0 +1,53 @@
+# Template file for 'coreboot-utils'
+pkgname=coreboot-utils
+version=4.8.1
+revision=1
+only_for_archs="i686 i686-musl x86_64 x86_64-musl"
+build_style=gnu-makefile
+wrksrc="coreboot-${version}"
+build_wrksrc="util"
+makedepends="pciutils-devel zlib-devel"
+short_desc="Coreboot firmware utilities"
+maintainer="m3tav3rse <n6maa10816@tuta.io>"
+license="GPL-2.0-only"
+homepage="https://coreboot.org"
+distfiles="https://coreboot.org/releases/coreboot-${version}.tar.xz>${pkgname}-\
+${version}.tar.xz"
+checksum=f0ddf4db0628c1fe1e8348c40084d9cbeb5771400c963fd419cda3995b69ad23
+
+post_extract() {
+	case "$XBPS_TARGET_MACHINE" in
```

Comment:
Why make the patching depend on the architecture? Do the patches break the build or hurt in any way when applied for a glibc build? If that's not the case, I would just put the patches in the patches directory and let xbps-src do the patching.

Reply:
You're ritght, it builds fine. I'll move this to patches